code/modules/uplink/uplink_items/uplink_explosives.dm

```diff
@@ -80,39 +33,129 @@
 	cost = 5
 	restricted = TRUE
 
+/datum/uplink_item/explosives/doorcharge
+	name = "Airlock Charge"
```

Comment:
No. Absolutely not. Airlock charge is not fun. Fuck off with this. Undetectable instant kills are shit.

Reply:
I agree with Chen, fuck the airlock charge it needed to die.

Reply:
Removed